### PR TITLE
Merge next into main for v6 Release

### PR DESCRIPTION
**Pull request by thomasdax98**
Migration Guide: https://github.com/vivid-planet/comet/pull/1629

### CODE REVIEW

File: .changeset/hungry-hornets-agree.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+"@comet/cms-admin": major
+"@comet/cms-api": major
+---
+
+Enhance CronJob module
```
Comment (thomasdax98) [inline .changeset/hungry-hornets-agree.md:6]: Added to migration guide

File: .changeset/dirty-trees-burn.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Add `MasterMenu` and `MasterMenuRoutes` components which both take a single data structure to define menu and routes.
```
Comment (thomasdax98) [inline .changeset/dirty-trees-burn.md:5]: @fraxachun TODO: Migration Guide

File: .changeset/odd-paws-speak.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+"@comet/admin": major
+---
+
+FinalForm: remove default `onAfterSubmit` implementation
```
Comment (thomasdax98) [inline .changeset/odd-paws-speak.md:5]: Added to migration guide

File: .changeset/poor-boxes-explain.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-icons": major
+---
+
+Remove deprecated icons `Betrieb`, `Logische Filter`, `Pool`, `Pool 2`, `State Green`, `State Green Ring`, `State Orange`, `State Orange Ring`, `State Red`, `State Red Ring`, `Vignette 1` and `Vignette 2`.
```
Comment (thomasdax98) [inline .changeset/poor-boxes-explain.md:5]: Added to migration guide

File: .changeset/rich-crews-smell.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+---
+"@comet/cms-admin": major
+---
+
+Make sites config generic
```
Comment (thomasdax98) [inline .changeset/rich-crews-smell.md:5]: Added to migration guide

File: .changeset/sharp-readers-tie.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/eslint-config": major
+---
+
+Enable `import/newline-after-import`
```
Comment (thomasdax98) [inline .changeset/sharp-readers-tie.md:5]: Added to migration guide

File: .changeset/sixty-cobras-brake.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+"@comet/cms-api": major
+---
+
+Replace ContentScopeModule with UserPermissionsModule
```
Comment (thomasdax98) [inline .changeset/sixty-cobras-brake.md:5]: @fraxachun TODO Migration Guide

File: .changeset/unlucky-files-compare.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/eslint-config": major
+---
+
+Enable no-other-module-relative-import rule by default
```
Comment (thomasdax98) [inline .changeset/unlucky-files-compare.md:5]: Added to migration guide

**Review by johnnyomair — DISMISSED** [review]

**Pull request merged**